src/components/VDiskPopup/VDiskPopup.tsx

```diff
@@ -97,20 +106,72 @@ const prepareVDiskData = (
         ReadThroughput,
         WriteThroughput,
         StoragePoolName,
+        Donors,
+        DonorMode,
+        Recipient,
     } = data;
 
-    const vdiskData: InfoViewerItem[] = [
-        {label: vDiskPopupKeyset('label_vdisk'), value: StringifiedId},
-        {
-            label: vDiskPopupKeyset('label_state'),
-            value: VDiskState ?? vDiskPopupKeyset('context_not-available'),
-        },
-    ];
+    const vdiskData: InfoViewerItem[] = [];
 
     if (StoragePoolName) {
         vdiskData.push({label: vDiskPopupKeyset('label_storage-pool'), value: StoragePoolName});
     }
 
+    if (Donors?.length && getVDiskLinkFn) {
```

Comment:
for now if `getVDiskLinkFn` is not defined, we won't show anything, but we could show list of id's in such case.